### PR TITLE
Update Swift documentation for latest version

### PR DESCRIPTION
**Pull request by nachoBonafonte**
### What does this PR do?
Update Swift documentation for latest version

### Motivation
We released a new version that adds new features that must be documented

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/continuous_integration/tests/swift.md [hunk 1]
```diff
@@ -302,6 +305,12 @@ For Network auto-instrumentation, you can configure these additional settings:
 `DD_ENABLE_NETWORK_CALL_STACK_SYMBOLICATED`
 : Shows the call stack information with not only the method name, but also the precise file and line information. May impact your tests' performance (Boolean)
 
+### Infrastructure Test correlation
+
+If you are running tests in your own infrastructure (mac or simulators tests), you can correlate your tests with your Infrastructure metrics reported by installing the Datadog agent and setting the following setting:
```
Comment (maycmlee) [inline content/en/continuous_integration/tests/swift.md:310]:
```suggestion
If you are running tests in your own infrastructure (MacOS or simulator tests), you can correlate your tests with your infrastructure metrics by installing the Datadog Agent and setting the following:
```

File: content/en/continuous_integration/tests/swift.md [hunk 2]
```diff
@@ -302,6 +305,12 @@ For Network auto-instrumentation, you can configure these additional settings:
 `DD_ENABLE_NETWORK_CALL_STACK_SYMBOLICATED`
 : Shows the call stack information with not only the method name, but also the precise file and line information. May impact your tests' performance (Boolean)
 
+### Infrastructure Test correlation
```
Comment (maycmlee) [inline content/en/continuous_integration/tests/swift.md:308]:
```suggestion
### Infrastructure test correlation
```

**Review by maycmlee — COMMENTED** [review]
Left some feedback!

**Comment by maycmlee** [comment]
Hi @nachoBonafonte! Just checking to see if these suggestion work for you?

**Comment by nachoBonafonte** [comment]
yes, they look perfect, except that I would use macOS instead of MacOS (Apple names it now lowercased)

**Review by maycmlee — APPROVED** [review]

**Pull request merged**